pkg/storegateway/series_refs.go

```diff
@@ -790,6 +795,39 @@ func seriesStreamingFetchRefsDurationIterator(iterator seriesChunkRefsSetIterato
 	})
 }
 
+// seriesIteratorStrategy defines the strategy to use when loading the series and their chunk refs.
+// See below for available options.
+type seriesIteratorStrategy byte
+
+const (
+	// By default, the strategy is to fetch series labels AND chunk refs
+	// for time ranges overlapping mint and maxt provided.
+	// To change the default behavior, use the flags below this.
+	defaultStrategy seriesIteratorStrategy = 0
+
+	// noChunks flag when used by itself fetches only series labels for series in the entire block.
+	noChunks seriesIteratorStrategy = 0b00000001
```

Comment:
```suggestion
	noChunkRefs seriesIteratorStrategy = 0b00000001
```

technically it's controlling the chunk refs, not the chunks